ReleaseNotes.txt

```diff
@@ -3,6 +3,9 @@ ASTRO - Release Notes:
 1.5 (Sprint 5):
 	- Functionality added to change my location and have the data update
 	- Input sanitized
+	- Project Presentation Finalized

```

Comment:
Could we remove this as this is not part of the applictation

Reply:
Also agree, the presentation is not related to the product, just a school assignment. Please remove :)